XenModel/XenSearch/Common.cs

```diff
@@ -646,7 +650,7 @@ private static IComparable MemoryTextProperty(IXenObject o)
                         vm.power_state != vm_power_state.Running)
                         return null;
 
-                    if (vm.virtualisation_status != VM.VirtualisationStatus.OPTIMIZED)
+                    if (!vm.GetVirtualisationStatus.HasFlag(VM.VirtualisationStatus.IO_DRIVERS_INSTALLED | VM.VirtualisationStatus.MANAGEMENT_INSTALLED))

```

Comment:
Good catch, I fixed these. There had been unnecessary checks for the presence of the Management Agent here.